package/crds/alibaba.crossplane.io_providerconfigs.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
were there any changes in these CRDs that you were worried about for breaking changes?

Reply:
Only the changes to Redis related CRDs will introduce breaking changes.
I tried with the existing implementation, but the older version of alicloud SDK returns error because multiple parameters have been deprecated.
I had to upgrade SDK version, which deprecated some old parameters and introduced new parameters for Redis provisioning.